src/codesy.coffee

```diff
@@ -5,7 +5,7 @@ codesy =
     form:
       heigth: 100
       width: 100
-    url: "https://api.codesy.io/"
+    url: "https://127.0.0.1:8443"
```

Comment:
Yeah, that sounds good.

I.e., keep `https://api.codesy.io` in the code, and then create a `gulp dev` task that replaces all api.codesy.io references with 127.0.0.1:8443 ?